docs/mpcnc/tops.md

```diff
@@ -0,0 +1,57 @@
+# Corner Tops and Rails
+
+Click on the images to embiggen.
```

Comment:
I've never been a big fan of this at the top of the pages. It just seems pretty obvious to me. I had to say it once, I'll be quiet about it now.